addon/hifi-connections/base.js

```diff
@@ -183,18 +185,19 @@ let Sound = Ember.Object.extend(Ember.Evented, DebugLogging, {
     let currentPosition = this._currentPosition();
     let newPosition     = (currentPosition + duration);
```

Comment:
in light of your sweet `Math.max` move, could you do `let newPosition = Math.min((currrentPosition + duration), audioLength);`?